packages/teams-js/test/public/runtime.spec.ts

```diff
@@ -132,31 +132,92 @@ describe('runtime', () => {
     });
   });
 
+  // Determines whether the given "subset" runtime object is a subset of the given "superset" runtime object.
+  // This is used to determine whether all capabilities supported in "subset" are also supported in "superset"
+  function isSubset(subset: object, superset: object): boolean {
+    for (const key in subset) {
+      if (typeof subset[key] === 'object' && typeof superset[key] === 'object') {
```

Comment:
I am not sure I follow how this test works, and when I try to paste this code into typescript playground it won't execute.

Reply:
Which test are you asking about? This function is just a helper and not a test in itself. The tests below leverage it.

I'm not sure why it wasn't working when you tried it, I pasted it into TS Playground with some sample objects and it seems to work [when I tried it](https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABDAzgZRAIxQUygChS1ygC5E5MArHaAGkSIAccAnE8ym6ASnMzhwANjgCGSAN4BYAFCJEwOK0T4ICFFEQBrHAE9kSItjw9E0ufOTAVUXSzjWjJANo7dAXUQBeH4gDkXLRQfogAZKGItvaOICzseK56nj5e-oHQfqbmlpYw1vgAhKgYxgROCW7uDMxsLpU8WbI5Oax4IKxIwKJCuADcTc0AvgPyg4g4PThWKjXxUIke3r7gACY4wDBgOCuNFi1tHQrdfSOIwxbn8q1Q7UhQrCA4-TLnsmpgGogPYLAAtjgARm8ZgGEAAFqIyGZBnRQd0hOQJDCBkxRABzHAoRGneRQUTYRHIi6wl6yN7qTTfP44ABMwOyiAg8MJJPkqIxWJBe0seIJ0NZZxJg2e5I+lPA1IAzPTTuzMdjuTlREwmAAhEBQKAIFmnImvGTvFDCHAAOiEcDR+AABsViHh8FSYP8aQxHf8AXxEAASCS20oOiVO2muwPuniDK08Z6G41mi3Wv0kAM-IOSkMpsPkH2J+1unBpr6hwHhyPPIA)